backend/test/todo.controller.js

```diff
@@ -0,0 +1,9 @@
+import chai from "chai";
+import request from "supertest";
+import Server from "../server";
+
+const expect = chai.expect;
+
+console.log(
```

Comment:
Try to at least add a stub test, where you test that the server and the new controller work.